src/plugins/vis_types/timeseries/server/lib/search_strategies/capabilities/default_search_capabilities.ts

```diff
@@ -77,12 +77,17 @@ export class DefaultSearchCapabilities {
     return this.createUiRestriction();
   }
 
+  public get whiteListedUIControls() {
+    return this.createUiRestriction();
+  }
+
   public get uiRestrictions() {
     return {
       [RESTRICTIONS_KEYS.WHITE_LISTED_METRICS]: this.whiteListedMetrics,
       [RESTRICTIONS_KEYS.WHITE_LISTED_GROUP_BY_FIELDS]: this.whiteListedGroupByFields,
       [RESTRICTIONS_KEYS.WHITE_LISTED_TIMERANGE_MODES]: this.whiteListedTimerangeModes,
-    };
+      [RESTRICTIONS_KEYS.WHITE_LISTED_UI_CONTROLS]: this.whiteListedUIControls,
```

Comment:
It seems a bit weird to work with "white listed ui controls" on the server - the other restriction keys in here seem rather specific, maybe it should be specific to the aggregate function? This seems like a more natural abstraction layer